packages/clay-table/src/__tests__/index.tsx

```diff
@@ -0,0 +1,227 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import * as React from 'react';
+import * as TestRenderer from 'react-test-renderer';
```

Comment:
I think we can only use `react-testing-library` from now on. What do you think @bryceosterhaus?

Reply:
Yeah I think it'd be a good rule of thumb to opt for always using `react-testing-library` and especially testing any functional parts of the component. I'm growing less and less confident in snapshots since its so easy to just "update all"

Reply:
Yep. I think a snapshot is mostly only useful as a smoke test (ie. did it render at all?) and as an *informational document of* (not a meaningful assertion about) what the component currently produces.